### PR TITLE
Polish Translations

### PR DESCRIPTION
**Pull request by AgentPiTi**
Done some of new Polish translations

**When merged this pull request will:**
- Describe what this pull request will do
- Each change in a separate line
- Include documentation if applicable
- Respect the [Development Guidelines](https://ace3mod.com/wiki/development/)

### CODE REVIEW

**Comment by jonpas** [comment]
@veteran29

**Comment by veteran29** [comment]
I would like to get #7374 merged first so I can check if these changes make sense. I don't want to check differences manually.

**Comment by veteran29** [comment]
Has conflicts, rebase?

**Comment by PabstMirror** [comment]
@veteran29 - Ready for review

File: addons/medical_gui/stringtable.xml [hunk 1]
```diff
@@ -985,7 +985,7 @@
             <Russian>Назотрахеальная трубка</Russian>
             <Spanish>Torniquete [CAT]</Spanish>
             <French>Canule Nasopharyngée [NPA]</French>
-            <Polish>Rurka nosowo-gardłowa [NPA]</Polish>
+            <Polish>Rurka nosowo-gardłowa [UG]</Polish>
```
Comment (veteran29) [inline addons/medical_gui/stringtable.xml]: UG = ustno gardlowa
```suggestion
            <Polish>Rurka nosowo-gardłowa [NPA]</Polish>
```

File: addons/medical_treatment/stringtable.xml [hunk 2]
```diff
@@ -453,6 +459,7 @@
             <Portuguese>Permite utilizar bolsas de IV para transfusão em si mesmo</Portuguese>
             <Chinese>啟用是否能對自己注射點滴</Chinese>
             <Czech>Umožňuje aplikovat IV transfuze na sama sebe.</Czech>
+            <Polish>Pozwala przetoczyć IV samemu sobie</Polish>
```
Comment (veteran29) [inline addons/medical_treatment/stringtable.xml]:
```suggestion
            <Polish>Pozwala przetoczyć płyny IV samemu sobie</Polish>
```

File: addons/medical_treatment/stringtable.xml [hunk 3]
```diff
@@ -2556,7 +2565,7 @@
             <English>Delayed</English>
             <Spanish>Retrasado</Spanish>
             <Russian>Срочная помощь</Russian>
-            <Polish>Opóźniony</Polish>
+            <Polish>ŻÓŁTY</Polish>
```
Comment (veteran29) [inline addons/medical_treatment/stringtable.xml]:
```suggestion
            <Polish>Pomoc Odroczona</Polish>
```

File: addons/medical_treatment/stringtable.xml [hunk 4]
```diff
@@ -2572,7 +2581,7 @@
             <English>Immediate</English>
             <Spanish>Inmediato</Spanish>
             <Russian>Неотложная помощь</Russian>
-            <Polish>Natychmiastowy</Polish>
+            <Polish>CZERWONY</Polish>
```
Comment (veteran29) [inline addons/medical_treatment/stringtable.xml]:
```suggestion
            <Polish>Pomoc Natychmiastowa</Polish>
```

File: addons/medical_treatment/stringtable.xml [hunk 5]
```diff
@@ -2588,7 +2597,7 @@
             <English>Deceased</English>
             <Spanish>Fallecido</Spanish>
             <Russian>Труп</Russian>
-            <Polish>Nie żyje</Polish>
+            <Polish>CZARNY</Polish>
```
Comment (veteran29) [inline addons/medical_treatment/stringtable.xml]:
```suggestion
            <Polish>Denat</Polish>
```

File: addons/medical_treatment/stringtable.xml [hunk 6]
```diff
@@ -2604,7 +2613,7 @@
             <English>None</English>
             <Spanish>Ninguno</Spanish>
             <Russian>Отсутствует</Russian>
-            <Polish>Brak</Polish>
+            <Polish>NIE SKLASYFIKOWANY</Polish>
```
Comment (veteran29) [inline addons/medical_treatment/stringtable.xml]:
```suggestion
            <Polish>Brak klasyfikacji</Polish>
```

**Pull request merged**